src/main/java/com/minecolonies/coremod/entity/ai/citizen/fisherman/EntityAIWorkFisherman.java

```diff
@@ -629,6 +609,27 @@ private void retrieveRod()
         entityFishHook = null;
     }
 
+    /**
+     * Let's the fisherman face the water, play the throw sound and create the fishingHook and throw it.
+     */
+    private void throwRod()
+    {
+        if (!world.isRemote)
+        {
+            WorkerUtil.faceBlock(job.getWater(), worker);
+            world.playSound((EntityPlayer) null,
+              this.worker.getPosition(),
+              SoundEvents.ENTITY_BOBBER_THROW,
+              SoundCategory.NEUTRAL,
+              0.5F,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.5F to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)